superset/db_engine_specs/bigquery.py

```diff
@@ -86,6 +93,13 @@ class BigQueryEngineSpec(BaseEngineSpec):
         "P1Y": "{func}({col}, YEAR)",
     }
 
+    custom_errors = {
+        INVALID_USER_PERMISSIONS: (
+            __("bad permissions"),
```

Comment:
```suggestion
            __("User does not have necessary permissions"),
```
How is this wording?